apps/super-app/src/common/Theme.ts

```diff
@@ -28,20 +28,25 @@ const callHomeTheme = createTheme(
   adaptV4Theme({
     typography: {
       h4: {
-        fontWeight: 600,
-      },
-      h5: {
         fontSize: 20,
         fontWeight: 700,
         color: '#133844',
         fontFamily: 'Hind',
         lineHeight: '28px',
       },
+      h5: {
+        fontSize: 18,
+        fontWeight: 600,
+        color: '#133844',
```

Comment:
I didn't realise that these were colour constants in the file. Could we use those values instead? e.g. `colors.text.primary` instead of `'#133844`. Having magic numbers makes their meaning difficult to understand, and this will make it easier to change the theme colours if we want.